composer.json

```
@@ -20,11 +20,32 @@
   },
   "require": {
     "php": ">=5.5",

```

```suggestion
    "php": ">=7.2.5",
```
we should use compatible version with Symfony 5.4

---

Upgraded the PHP version to 7.2.5